core/types/transaction_marshalling.go

```diff
@@ -56,6 +56,10 @@ type txJSON struct {
 
 	// Only used for encoding:
 	Hash common.Hash `json:"hash"`
+
+	ParentHash *common.Hash    `json:"parentHash,omitempty"`
```

Comment:
Does tx unmarshalling not work? I have been using it in ethclient without issues. Ideally we remove these fields entirely

Reply:
it doesnt work right now for some blocks